site_scons/components/__init__.py

```diff
@@ -301,9 +301,9 @@ def define_components(reqs):
     reqs.define('fuse', libs=['fuse3'], defines=['FUSE_USE_VERSION=35'],
                 retriever=GitRepoRetriever('https://github.com/libfuse/libfuse.git'),
                 commands=[['meson', 'setup', '../fuse'],
-                          ['meson', 'configure', '--prefix=$FUSE_PREFIX', '-Ddisable-mtab=True',
-                           '-Dudevrulesdir=$FUSE_PREFIX/udev', '-Dutils=False',
-                           '--default-library', 'both'],
+                          ['meson', 'configure', '--prefix', '$FUSE_PREFIX', '-D',
```

Comment:
Where does FUSE_PREFIX get set?

Reply:
in site_scons/prereq_tools/base.py _modify_prefix

`            self._save_component_prefix(f'{comp_def.name.upper()}_PREFIX', '/usr')`

Reply:
actually, in this case, it would be in get_prefixes

        self._save_component_prefix(comp_prefix, target_prefix)